Assets/Scripts/Cards/ChaosCard.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "New Chaos Power Card", menuName = "Chaos Power Card")]
+public class ChaosCard:Card {
+
+	private override bool targetable = true;
```

Comment:
Nit: I like to read boolean variable names with prepositions - e.g. `isTargetable`